lib/repo-audit/checks/at_least_one_file_check.rb

```diff
@@ -2,7 +2,7 @@ module RepoAudit::Checks
   class AtLeastOneFileCheck < BaseCheck
     include RepoAudit::RepositoryContent
 
-    register_check :at_least_one_file_check
+    register_check :at_least_one_file
```

Comment:
In hindsight, I wonder if this should have been called `has_continuous_integration_configured` or something? 🤔